subsystem/tests/helpers/mod.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://spdx.org/licenses/MIT
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author(s): L. Kuklinek
+
+static INIT: std::sync::Once = std::sync::Once::new();
+
+pub fn init_test_runtime() -> tokio::runtime::Runtime {
```

Comment:
This can be pub(crate)

Reply:
Frankly, I don't see the point. This is not a library so will not be "imported" elsewhere. In addition to that, it is in a private module so would not be accessible if you imported it as a library anyway.